CHANGELOG.md

```diff
@@ -1,3 +1,64 @@
+# dbt_zendesk v0.25.0
+
+[PR #204](https://github.com/fivetran/dbt_zendesk/pull/204) includes the following updates:
+
+## Schema/Data Changes
+
+**2 total changes • 2 possible breaking changes**
+| **Data Model/Column** | **Change type** | **Old** | **New** | **Notes** |
+| -------------- | --------------- | ------------ | ------------ | --------- |
+| [`zendesk__ticket_metrics`](https://fivetran.github.io/dbt_zendesk/#!/model/model.zendesk.zendesk__ticket_metrics)<br>• `first_resolution_business_minutes`<br>• `first_reply_time_business_minutes`<br>• `agent_wait_time_in_business_minutes`<br>• `requester_wait_time_in_business_minutes`<br>• `solve_time_in_business_minutes`<br>• `agent_work_time_in_business_minutes`<br>• `on_hold_time_in_business_minutes`<br>• `new_status_duration_in_business_minutes`<br>• `open_status_duration_in_business_minutes`<br>• `first_reply_time_calendar_minutes`<br>• `total_reply_time_calendar_minutes`<br>• `ticket_unassigned_duration_calendar_minutes`<br>• `requester_last_login_age_minutes`<br>• `assignee_last_login_age_minutes`<br>• `unsolved_ticket_age_minutes`<br>• `unsolved_ticket_age_since_update_minutes` | Column Data Type | Integer | Numeric rounded to 4th decimal place | Removed logic casting ticket metric values as integers in intermediate models. This may have caused minor rounding discrepancies between the Zendesk data models and UI reports. ([PR #204](https://github.com/fivetran/dbt_zendesk/pull/204)) |
+| [`zendesk__sla_policies`](https://fivetran.github.io/dbt_zendesk/#!/model/model.zendesk.zendesk__sla_policies)<br>• `sla_elapsed_time` | Data Type | Integer | Numeric rounded to 4th decimal place | May particularly affect `agent_work_time`, `first_reply_time`, `next_reply_time`, and `requester_wait_time` SLA policy metrics. ([PR #204](https://github.com/fivetran/dbt_zendesk/pull/204)) |
+
+### Under the Hood
+
+[PR #205](https://github.com/fivetran/dbt_zendesk/pull/205) includes the following updates:
+
+- Updated conditions in `.github/workflows/auto-release.yml`.
+- Added `.github/workflows/generate-docs.yml`.
+- Added `+docs: show: False` to `integration_tests/dbt_project.yml`.
+- Migrated `flags` (e.g., `send_anonymous_usage_stats`, `use_colors`) from `sample.profiles.yml` to `integration_tests/dbt_project.yml`.
+- Updated `maintainer_pull_request_template.md` with improved checklist.
+- Refreshed README tag block:
+  - Standardized Quickstart-compatible badge set
+  - Left-aligned and positioned below the H1 title.
+- Updated Python image version to `3.10.13` in `pipeline.yml`.
+- Added `CI_DATABRICKS_DBT_CATALOG` to:
+  - `.buildkite/hooks/pre-command` (as an export)
+  - `pipeline.yml` (under the `environment` block, after `CI_DATABRICKS_DBT_TOKEN`)
+- Added `certifi==2025.1.31` to `requirements.txt` (if missing).
```

Comment:
can remove these

Reply:
Are these not applicable/relevant, or just too detailed to include in the CHANGELOG?